styles.css

```diff
@@ -0,0 +1,63 @@
+* {
+    font-family: 'Roboto', sans-serif;
+}
+
+main {
```

Comment:
I think you can check the styling applied to the main section since it seems they aren't being applied properly on bigger screens.

![image](https://user-images.githubusercontent.com/107666488/212735628-79b63b90-8643-4147-9e48-a521dc7fdd85.png)